src/web/styles/themes.scss

```diff
@@ -30,8 +30,10 @@
   --theme-warning-background: #fefcd3;
   --theme-info-icon: #7085d4;
   --theme-error-icon: #e52900;
+  --theme-error-border: #e52900;
```

Comment:
I do wonder if this colour would ever realistically differ from `--theme-error-icon`? 
Maybe we could simplify and just have `theme-error: #e52900;` (i.e. change from `red`) and `--theme-error-background: #facec5;` and remove the specific `icon` and `border` ones.

What do you think?

Reply:
make sense, changed